cfgov/regulations3k/models/pages.py

```diff
@@ -592,3 +590,16 @@ def validate_page_number(request, paginator):
     except InvalidPage:
         page_number = 1
     return page_number
+
+
+def validate_regs_list(request):
+    """
+    A utility for validating a RegulationsSearchPage request.
+
+    Validates that a list of regulation part numbers is alphanumeric.
+    """
+    if 'regs' in request.GET and request.GET.get('regs'):
+        regs_input_list = request.GET.getlist('regs')
+        return [reg for reg in regs_input_list if reg.isalnum()]
+    else:
```

Comment:
I think to get the code coverage up we'll need to test this `'regs' not in request.GET` condition.

Reply:
Good call. Added that test case.